socialhome/users/views.py

```diff
@@ -67,18 +68,11 @@ def get_context_data(self, **kwargs):
 
     def get_json_context(self):
         json_context = super().get_json_context()
-        json_context.update({
-            "profile": {
-                "id": self.target_profile.id,
-                "guid": self.target_profile.guid,
-                "followersCount": self.followers_count,
-                "followingCount": str(self.target_profile.following.count()),
-                "handle": self.target_profile.handle,
-                "saferImageUrlLarge": self.target_profile.safer_image_url_large,
-                "streamType": self.profile_stream_type,
-                "pinnedContentExists": self.pinned_content_exists,
-            },
-        })
+        if self.request.user.is_authenticated:
```

Comment:
This should be removed AFAICT - we always want to serialize the profile.

Reply:
Yes, that's right.